Stage I/Stage I.py

```diff
@@ -40,16 +53,25 @@
 historical2 = historical2.drop(historical2.columns[4:],axis = 1)
 historical2 = historical2.drop(historical2.columns[0],axis = 1)
 
+# Here I would recomment splitting these operations over multiple lines using
+# the &= operator, so something like
+# a = b & c
+# a &= d
+# a &= e
+# instead of a = b & c & d & e
```

Comment:
This made sense to me before, but I got confused when I tried to modify accordingly. For example, pop_dep2 is kind of like a = [b, c], and b = x[m & n].sum(), c = x[h & n].sum(). So it doesn't seem to me the same as a = b & c & e & d &e. Could you explain a bit more?